feature/google/src/main/java/com/revenuecat/purchases/google/storeProductConversions.kt

```diff
@@ -22,5 +23,48 @@ fun SkuDetails.toStoreProduct() =
         introductoryPricePeriod.takeIf { it.isNotBlank() },
         introductoryPriceCycles,
         iconUrl,
-        JSONObject(originalJson)
+        JSONObject(originalJson),
+        getPricingPhases()
     )
+
+fun SkuDetails.getPricingPhases(): List<PricingPhase> {
+    if (subscriptionPeriod.isBlank()) {
+        return emptyList() // TODO what should we do for one time purchases?
+    }
+    val phases = mutableListOf<PricingPhase>()
+    if (freeTrialPeriod.isNotBlank()) {
+        phases.add(
+            PricingPhase(
+                freeTrialPeriod,
+                1,
+                "Free",
+                0,
+                priceCurrencyCode,
+                PricingPhase.FINITE_RECURRING
```

Comment:
did we see how pricing phases are made for migrated subs to get at these values? or was this just kinda a smart estimate of what a pricing phase would look like?

Reply:
I haven't checked a migrated subscription, but I checked the one I created with a free trial and it looks like this:

```
      "pricingPhases": [
        {
          "priceAmountMicros": 0,
          "priceCurrencyCode": "EUR",
          "formattedPrice": "Free",
          "billingPeriod": "P5D",
          "recurrenceMode": 2,
          "billingCycleCount": 1
        },
        {
          "priceAmountMicros": 2090000,
          "priceCurrencyCode": "EUR",
          "formattedPrice": "€2.09",
          "billingPeriod": "P1M",
          "recurrenceMode": 1
        }
      ],
```